imgui/src/context.rs

```diff
@@ -270,6 +276,7 @@ pub struct SuspendedContext(Context);
 
 impl SuspendedContext {
     /// Creates a new suspended imgui-rs context.
+    #[doc(alias = "CreateContext")]
```

Comment:
For now, let's keep the SuspendedContext out of the doc comments. Its current implementation is wrapped up in our own single-threaded safety guarantees, and not necessarily in ImGui's create context. I don't want to confuse someone who searched "CreateContext" and sees something like this in docs.rs